framework/include/convergence/Convergence.h

```diff
@@ -58,6 +60,9 @@ class Convergence : public MooseObject,
   /// Performance ID for \c checkConvergence
   PerfID _perfid_check_convergence;
 
+  /// Thread ID
+  THREAD_ID _tid;
```

Comment:
are convergence objects threaded?

Reply:
as in is there one copy per thread?

Reply:
Yes, if I understand correctly that this is proof:
```
void
FEProblemBase::addConvergence(const std::string & type,
                              const std::string & name,
                              InputParameters & parameters)
{
  parallel_object_only();

  for (THREAD_ID tid = 0; tid < libMesh::n_threads(); tid++)
  {
    std::shared_ptr<Convergence> conv = _factory.create<Convergence>(type, name, parameters, tid);
    _convergences.addObject(conv, tid);
  }
}
```

Reply:
yup